packages/terra-collapsible-menu-view/src/terra-dev-site/doc/collapsible-menu-view/AlwaysCollapsedMenuItems.2.doc.mdx

```diff
@@ -0,0 +1,31 @@
+import { Badge } from 'terra-collapsible-menu-view/package.json?dev-site-package';
+
+import CollapsibleMenuViewProps from 'terra-collapsible-menu-view/src/CollapsibleMenuView?dev-site-props-table';
+import CollapsibleMenuViewItemProps from 'terra-collapsible-menu-view/src/CollapsibleMenuViewItem?dev-site-props-table';
+import CollapsibleMenuViewItemGroupProps from 'terra-collapsible-menu-view/src/CollapsibleMenuViewItemGroup?dev-site-props-table';
+import CollapsibleMenuViewToggleProps from 'terra-collapsible-menu-view/src/CollapsibleMenuViewToggle?dev-site-props-table';
+
+import AlwaysCollapsedMenuItemsDemo from "../example/AlwaysCollapsedMenuItemsDemo";
+
+<Badge />
+
+# Terra Collapsible Menu View with always shown collapsed menu items
+
+- For Collapsible menu view, you can use `alwaysCollapsedMenuItems` attribute in `CollapsibleMenuView` to pre-populate collapsible menu view with menu items.
+- These items will always hidden under the More button. The collapsed menu view will always shown if at least one item is added to the collapsedMenuItems list. A divider will shown if there are other elements pushed to the collapsed menu view to divide them from the prepopulated list.
+- For base usage, see the Collapsible menu view main documentation
+
+## Examples
+<AlwaysCollapsedMenuItemsDemo title='Always collapsed menu items' />
```

Comment:
@EPAMMarcell - I think there was a misunderstanding. We don't need a new full doc page just for this new prop, we just need a new separated example added to the current first/main doc page.

You can remove this file `AlwaysCollapsedMenuItems.2.doc.mdx` entirely. 

Instead add your new example to `CollapsibleMenuView.1.doc.mdx` in addition to the current one:
```diff
   ## Examples
   <BasicView title='CollapsibleMenuView Example' />
+  <AlwaysCollapsedMenuItemsDemo title='Always collapsed menu items' />
```

\
\
You can also add a new line (would be the 6th paragraph) to the description at the top of `CollapsibleMenuView.1.doc.mdx` similar to what you had here, suggestion is below. Plus, you could fix two typos that are currently there I just noticed ( "_An Item group that is not_ **slectable** _will have_ **it's** _children roll ..._" ) while you are making the changes:
```diff
-  CollapsibleMenuView.ItemGroup will flex between a button group when displayed face-up and a Menu.ItemGroup when rolled into the ellipsis menu. A selectable ItemGroup will have toggle style selection. An Item group that is not slectable will have it's children roll into the ellipsis menu as non selectable Menu.Items.

+  CollapsibleMenuView.ItemGroup will flex between a button group when displayed face-up and a Menu.ItemGroup when rolled into the ellipsis menu. A selectable ItemGroup will have toggle style selection. An Item group that is not selectable will have its children roll into the ellipsis menu as non selectable Menu.Items.

CollapsibleMenuView.Divider will flex between a vertical divider when displayed face-up and a Menu.Divider when in a menu.

+ To pre-populate the collapsible menu view with menu items that always remain collapsed, use the `alwaysCollapsedMenuItems` prop.

## Getting Started
```

Reply:
Neil, you are right, there were some misunderstanding I tried to follow the pattern, and saw that other components has different demo pages. No problem, moved it back in the right place :)
Fixed the typos as well. :) Still working on the wdio tests - it seems like somehow what I run on mac is differs from what the travisCI runs. One of my co-worker will update the screenshots soon.